CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
   - xSQLServerAlwaysOnAvailabilityGroup
 - Changes to xSQLServerSetup
   - Properly checks for use of SQLSysAdminAccounts parameter in $PSBoundParameters. The test now also properly evaluates the setup argument for SQLSysAdminAccounts.
+- Changes to xSQLServerSetup
```

Comment:
You can remove this line. It's already a row "Changes to xSQLServerSetup" a little bit above.